modules/exploits/x3/x3_adxsrv_cmd.rb

```diff
@@ -0,0 +1,347 @@
+#
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::EXE
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Sage X3 Administration Service Command Execution',
+        'Description' => %q{
+          This module leverages an authentication bypass exploit within Sage X3 AdxSrv's administration
+          protocol to execute arbitrary commands as SYSTEM against a Sage X3 Server running an
+          available AdxAdmin service.
+        },
+        'Author' => [
+          'Jonathan Peterson <deadjakk[at]shell.rip>', # @deadjakk
+          'Aaron Herndon' # @ac3lives
+        ],
+        'License' => MSF_LICENSE,
+        'DisclosureDate' => '2020-07-07',
```

Comment:
Presumably this should be `2021`.

```suggestion
        'DisclosureDate' => '2021-07-07',
```